src/WebJobs.Script/RuntimeInformation/SystemRuntimeInformation.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class SystemRuntimeInformation : ISystemRuntimeInformation
+    {
+        public Architecture GetOSArchitecture()
+        {
+            return RuntimeInformation.OSArchitecture;
+        }
+
+        public OSPlatform GetOSPlatform()
+        {
+            // Default to Linux
+            OSPlatform os = OSPlatform.Linux;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                os = OSPlatform.Windows;
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
After our offline discussion, this helper is very specific to figuring out worker path for a OS and not a generic GetOSPlatform. Move this under Rpc namespace and use default OS specified in the the worker.config . If not default specified throw